beesdoo_easy_my_coop/models/subscription_request.py

```diff
@@ -1,7 +1,9 @@
 # Copyright 2019 Coop IT Easy SC
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import fields, models
+from odoo import api, fields, models
+
+_MAJORITY = 18
```

Comment:
Should this be a configurable parameter?

Reply:
It could be but I won't add now 😇